etl/churn.kp/knowledge.md

```diff
@@ -455,6 +464,9 @@ def compute_churn_week(df, week_start, bucket, prefix):
           .filter(df['subsession_start_date'] >= week_start_hyphenated)
           .filter(df['subsession_start_date'] < week_end_excl)
     )
+    
+    # clean some of the aggregate fields
+    current_week = current_week.na.fill(0, ["total_uri_count", "unique_domains_count"])
```

Comment:
The mean of the `unique_domains_count` will probably suffice for this application.